tools/pot/setup.py

```diff
@@ -65,15 +65,15 @@ def get_version():
     sys.argv.remove('--install-dev-extras')
 
 INSTALL_REQUIRES = [
+    "numpy>=1.16.6,<=1.23.1; python_version < '3.11'",
```

Comment:
Does POT not support 1.24 NumPy? Can you please tell why?

Reply:
I probably didn't get your message. The problem here is that there is no rebuild numpy 1.24 package for python_version < '3.11' which can cause the issue on the user's side.